hack/update-mocks.sh

```diff
@@ -45,22 +35,26 @@ pushd "${KUBE_ROOT}/hack/tools" >/dev/null
 popd >/dev/null
 
 function git_find() {
-    # Similar to find but faster and easier to understand.  We want to include
```

Comment:
You should send these as a different PR :)

Reply:
fair!